admin_guide/install/advanced_install.adoc

```diff
@@ -83,22 +83,71 @@ the remaining instructions.
 
 == Configuring Ansible
 
-You must create an *_/etc/ansible/hosts_* file, which is Ansible's inventory
-file for the playbook to use during the installation.
+The *_/etc/ansible/hosts_* file is Ansible's inventory
+file for the playbook to use during the installation. Replace the contents of the file with your desired configuration.
 
-Any hosts you designate as masters during the installation process should also
-be configured as
+[NOTE]
+====
+Before running the Ansible installer, any hosts you intend to designate as
+masters during the installation process should also be configured as
 link:../../admin_guide/manage_nodes.html#marking-nodes-as-unschedulable-or-schedulable[unschedulable]
-nodes. This is so the masters are configured as part of the
+nodes, in order to configure the masters as part of the
 link:../../architecture/additional_concepts/networking.html#openshift-sdn[OpenShift
 SDN].
+====
 
-The following sections describe example inventory files for various environment
-topographies, including configuring
-link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#high-availability-masters[multiple
+The following sections describe example *_/etc/ansible/hosts_* files for various environment
+topographies, including example configurations for
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#high-availability-masters[using multiple
 masters for high availability]. You can choose an example that matches your
-requirements to modify and use as your inventory file when
-link:#running-the-ansible-installer[running the Ansible installer].
+requirements and use as your inventory file when
+link:#running-the-ansible-installer[running the Ansible installer], or modify an example to match your own environment.
+
+*Configuring Node Host Labels*
+
+To assign link:../../architecture/core_concepts/pods_and_services.html#labels[labels] to node hosts during the Ansible install, indicate the desired labels in the *_/etc/ansible/hosts_* file:
```

Comment:
(Suggestion) May be overkill, but possible rework to be more explicit and provide explanation first:

"You can assign labels to node hosts during the Ansible install using the the /etc/ansible/hosts file. Labeling nodes is useful for determining the placement of pods onto nodes using the scheduler.

To do so, use the openshift_node_labels variable to indicate the desired labels and add it to the end of the desired host entry in the [nodes] section:"